src/Language/Printer.php

```diff
@@ -181,12 +182,12 @@ public function printAST($ast)
                             . $node->selectionSet;
                     },
 
-                    NodeKind::INT => static function (IntValueNode $node) {
-                        return $node->value;
+                    NodeKind::INT => static function (IntValueNode $node) : string {
+                        return strval($node->value);
```

Comment:
In theory `IntValueNode->value` should be always a string. This change won't harm but it may simply mean that the error is somewhere else (specifically when constructing an instance of `IntValueNode`

Reply:
Yes, true. I'll investigate, but it would be great if you could merge this in the meantime, as it is blocking myself and another developer.